frontend/src/modules/organization/components/details/identity/organization-details-identity-item.vue

```diff
@@ -0,0 +1,204 @@
+        <lf-icon
+          name="fingerprint-fill"
+          :size="20"
+          class="text-gray-600"
+        />
+      </lf-tooltip>
+    </div>
+
+    <div class="pl-3 flex-grow">
+      <div class="flex items-center">
+        <div class=" flex items-center">
+          <p v-if="!props.identity.url" class="text-medium max-w-48 truncate">
+            {{ props.identity.value }}
+          </p>
+          <a
+            v-else
+            :href="props.identity.url"
+            target="_blank"
+            rel="noopener noreferrer"
+            class="text-medium cursor-pointer !text-black underline decoration-dashed
+             decoration-gray-400 underline-offset-4 hover:decoration-gray-900 max-w-48 truncate"
+          >
+            {{ props.identity.value }}
+          </a>
+        </div>
+        <lf-tooltip v-if="props.identity.verified" content="Verified identity">
+          <lf-icon
+            name="verified-badge-line"
+            :size="16"
+            class="ml-1 text-primary-500"
+          />
+        </lf-tooltip>
+      </div>
+    </div>
+
+    <!-- Dropdown -->
+    <lf-dropdown v-if="hovered && hasPermission(LfPermission.organizationEdit)" placement="bottom-end" width="232px">
+      <template #trigger>
+        <lf-button type="secondary-ghost" size="small" :icon-only="true">
+          <lf-icon name="more-fill" />
+        </lf-button>
+      </template>
+      <!-- Edit identity -->
+      <lf-dropdown-item
+        class="w-full"
+        @click="emit('edit')"
+      >
+        <lf-icon name="pencil-line" />Edit identity
+      </lf-dropdown-item>
+
+      <lf-dropdown-separator />
+
+      <!-- Verified -->
+      <lf-tooltip
+        v-if="props.identity.verified"
+        placement="top"
+        :disabled="!isVerifyDisabled"
+        class="!w-full"
+      >
+        <template #content>
+          Identities tracked from an Integrations<br> can’t be unverified
+        </template>
+        <lf-dropdown-item
+          v-if="props.identity.verified"
+          placement="top"
+          :disabled="isVerifyDisabled"
+          class="w-full"
+          @click="verifyIdentity(false)"
+        >
+          <lf-svg name="unverify" class="!h-4 !w-4 text-gray-600" />Unverify identity
+        </lf-dropdown-item>
+      </lf-tooltip>
+      <lf-tooltip
+        v-else
+        placement="top"
+        :disabled="!isVerifyDisabled"
+        class="!w-full"
+      >
+        <template #content>
+          Identities tracked from an Integrations<br> can’t be verified
+        </template>
+        <lf-dropdown-item
+          :disabled="isVerifyDisabled"
+          class="w-full"
+          @click="verifyIdentity(true)"
+        >
+          <lf-icon name="verified-badge-line" />Verify identity
+        </lf-dropdown-item>
+      </lf-tooltip>
+
+      <!-- Unmerge -->
+      <lf-dropdown-item @click="emit('unmerge')">
+        <lf-icon name="link-unlink" />Unmerge identity
```

Comment:
If you try to unmerge a LinkedIn identity it throws an error. It seems that it's not sending the payload.
Try here https://lf-staging.crowd.dev/organizations/1544ab80-da1e-11ee-85d4-778cfceb14e0?projectGroup=901e303b-5188-4338-9be8-dd96f45d7eca&segmentId=901e303b-5188-4338-9be8-dd96f45d7eca

It could be because in CrowdDotDev linkedin is added as company:CrowdDotDev as expected. And for instance in cal.com since I added it manually https://lf-staging.crowd.dev/organizations/193e0e50-da20-11ee-85d4-778cfceb14e0?projectGroup=901e303b-5188-4338-9be8-dd96f45d7eca&segmentId=901e303b-5188-4338-9be8-dd96f45d7eca, the identity was created without the company: prefix